components/SignUpForm.tsx

```diff
@@ -1,287 +1,128 @@
-import { Form, Input, message, Radio } from 'antd';
-import 'antd/dist/antd.css';
+import { Radio } from 'antd';
 import { RadioChangeEvent } from 'antd/lib/radio';
 import axios from 'axios';
+
 import 'cleave.js/dist/addons/cleave-phone.us';
 import Cleave from 'cleave.js/react';
 import React, { useState } from 'react';
-import QuestionController from './QuestionController';
+import games from '../lib/games';
+import Question from './Question';
+interface SignUpFormProps {
+  gameID: string;
+}
 
-const SignUpForm: React.FC = () => {
+const SignUpForm: React.FC<SignUpFormProps> = ({ gameID }) => {
   const [name, setName] = useState<string>('');
   const [email, setEmail] = useState<string>('');
-  const [city, setCity] = useState<string>('');
   const [handle, setHandle] = useState<string>('');
-  const [question, setQuestion] = useState<number>(0);
   const [answer, setAnswer] = useState<string>('');
   const [phoneNumber, setPhoneNumber] = useState<string>('');
-  const [wager, setWager] = useState<number>(1);
-  const [selected, setSelected] = useState<number>(1);
-  const changeGame = () => {
-    setSelected(1);
-  };
+  const [wager, setWager] = useState<number>();
```

Comment:
do you want a default value for wager? maybe 1

Reply:
Sure I’ll add that